en/PlanView/PlanView.md

```diff
@@ -108,22 +115,76 @@ The list of commands displayed in the dialog can be filtered by category. For ex
 > **Note** The list of available commands will depend on firmware and vehicle type. Examples may include: Waypoint, Start image capture, Jump to item (to repeat mission) and other commands.
 
 
-### Mission Start
+### Mission Settings {#mission_settings}
+
+The *Mission Start* panel  is the first item that appears in the [mission command list](#mission_command_list).
+It may be used to specify a number default settings that may affect the start or end of the mission. 
+
+![Mission Command List - showing mission settings](../../assets/plan/mission_start.png)
+
+![Mission settings](../../assets/plan/MissionSettings.png)
+
+#### Mission Defaults
+
+##### Waypoint alt
+
+Set the default altitude for the first mission item added to a plan (subsequent items take an initial altitude from the previous item). 
+This can also be used to change the altitude of all items in a plan to the same value; you will be prompted if you change the value when there are items in a plan.
+
+##### Flight speed
+
+Set a flight speed for the mission that is different than the default mission speed.
+
+#### Mission End
+##### Return to Launch after mission end
+
+Check this if you want your vehicle to Return/RTL after the final mission item.
+
+
+#### Planned Home Position
+
+The [Planned Home Position](#planned_home) section allows you to simulate the vehicle's home position while planning a mission. 
+This allows you to view the waypoint trajectory for your vehicle from takeoff to mission completion. 
+
+![MissionSettings Planned Home Position Section](../../assets/plan/MissionSettingsPlannedHomePositionSection.jpg)
+
+> **Note** This is only the *planned* home position and you should place it where you plan to start the vehicle from. 
+  It has no actual impact on flying the mission. 
+  The actual home position of a vehicle is set by the vehicle itself when arming.
+
+The section allows you to set the **Altitude** and **Set Home to Map Centre** 
+(you can move it to another position by dragging it on the map).
+
+
+
+#### Camera
+
+The camera section allows you to specify a camera action to take, control the gimbal and set your camera into photo or video mode.
+
+![MissionSettings Camera Section](../../assets/plan/MissionSettingsCameraSection.jpg)
+
+The available camera actions  are:
+
+*  Continue current action 
```

Comment:
@DonLakeFlyer This didn't appear to be an option. Is it conditional on something? (this comes from release notes).

Reply:
Should be: "No change (continue current action)" or something like that.

Reply:
The combo box item ius "No change"

Reply:
Cheers. Fixed in https://github.com/mavlink/qgc-user-guide/commit/350738f7e1583aeadc5a0e0f0477649bde4428fb